app/uk/gov/hmrc/thirdpartyapplication/services/SubscriptionService.scala

```diff
@@ -46,18 +45,12 @@ class SubscriptionService @Inject()(applicationRepository: ApplicationRepository
     subscriptionRepository.searchCollaborators(context, version, partialEmailMatch)
   }
 
-
   def fetchAllSubscriptions(): Future[List[SubscriptionData]] = subscriptionRepository.findAll()
 
   def fetchAllSubscriptionsForApplication(applicationId: UUID)(implicit hc: HeaderCarrier): Future[List[ApiSubscription]] = {
-    def fetchApis: Future[List[ApiDefinition]] = apiDefinitionConnector.fetchAllAPIs(applicationId) map {
-      _.map(api => api.copy(versions = api.versions.filterNot(_.status == ALPHA)))
-        .filterNot(_.versions.isEmpty)
```

Comment:
Do we still need the the filtering out of empty versions?

Reply:
I don't think we do - it was there to stop us returning contexts with no versions when we had filtered out all the ALPHA versions.

We really shouldn't have any contexts without versions so the filter is not doing much.